csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -34,7 +34,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" Version="1.0.0"/>
+    <!-- <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" Version="1.0.0"/> -->
```

Comment:
Unclear why this is commented out. If it's no longer needed let's remove it, otherwise we can uncomment it back.

Reply:
Uncommented it. 
This was creating build error in local, so added a dotnet workflow for build and test and it works fine.